trySwift/ScheduleViewController.swift

```diff
@@ -30,8 +30,6 @@ class ScheduleViewController: ButtonBarPagerTabStripViewController {
         buttonBarView.backgroundColor = .white
         settings.style.selectedBarBackgroundColor = .white
         buttonBarView.selectedBar.backgroundColor = .trySwiftAccentColor()
-        
-        tabBarController?.delegate = self
```

Comment:
I removed this since it's handled at a higher level now (RootTabBarController delegates this functionality via protocol conformance now)